client/src/store/graphql/utils/Filters.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import FilterContainer from '@ferlab/ui/core/components/filters/FilterContainer';
+import FilterSelector from '@ferlab/ui/core/components/filters/FilterSelector';
+import { IFilter, IFilterGroup } from '@ferlab/ui/core/components/filters/types';
+import { keyEnhance, keyEnhanceBooleanOnly, underscoreToDot } from '@ferlab/ui/core/data/arranger/formatting';
+import {
+  getFilterType,
+  getSelectedFilters,
+  updateFilters,
+} from '@ferlab/ui/core/data/filters/utils';
+import * as H from 'history';
+
+import { ExtendedMappingResults } from 'store/graphql/prescriptions/actions';
+
+import { GQLData, Results } from '../prescriptions/models';
+
+export interface RangeAggs {
+  stats: {
+    max: number;
+    min: number;
+  };
+}
+
+export interface TermAggs {
+  buckets: TermAgg[];
+}
+
+export interface TermAgg {
+  doc_count: number;
+  key: string;
+}
+
+export type Aggs = TermAggs | RangeAggs;
+
+const isTermAgg = (obj: any): obj is TermAggs => obj.buckets !== undefined;
+const isRangeAgg = (obj: any): obj is RangeAggs => obj.stats !== undefined;
+
+export type ExtendedMapping = {
+  displayName: string;
+  field: string;
+  isArray: boolean;
+  type: string;
+  rangeStep?: number;
+};
+
+//TODO investigate: should only be called once per tab.
```

Comment:
I think we should create a ticket instead and remove the comment. And why not add it a the git commit message instead?